src/lib/stores/QTheme.ts

```diff
@@ -69,10 +79,9 @@ async function prepareThemeColors(from: string) {
   return themeColors;
 }
 
-const defaultThemeColors = await prepareThemeColors("#3499E7");
-
 function themeBuilder() {
-  const { subscribe, set, update } = writable(defaultThemeColors);
+  const { subscribe, set, update } = writable({} as Awaited<ReturnType<typeof prepareThemeColors>>);
```

Comment:
Quite ugly solution indeed. We might really want to change this in the future. Could you open an issue?